app/index.js

```diff
@@ -83,31 +83,31 @@ module.exports = class GeneratorLicense extends Generator {
         name: 'name',
         message: 'What\'s your name:',
         default: this.options.name || this.gitc.user.name,
-        when: this.options.name == null
+        when: !this.options.name
```

Comment:
I know. In these cases I'd like to have something like `when:`, but `while:`, so you could throw in some validation, for when you need output. What good does an empty name do in cases as such? That other generator shouldn't even call this sub-generator if name is not set (and he does not want to ask it again). Otherwise, we are allowing name to be optional, and then rather adjust the templates and options, to let users know that it's optional.